backend/evoucher/settings/base.py

```diff
@@ -139,6 +119,12 @@
 
 USE_TZ = True
 
+#Email Configuration
+EMAIL_HOST = "smtp.gmail.com"
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+EMAIL_HOST_USER = "nussu.evoucher.dev@gmail.com"
+EMAIL_HOST_PASSWORD = "vzdceuzcmayocdqk"
```

Comment:
For security reasons, it would be better to store the password (and optional the user as well) in a `.env` file, so that other people will not be able to see what the password is, since this repository is public.

Reference: https://stackoverflow.com/questions/62925571/how-do-i-use-env-in-django